### PR TITLE
[FND-3088] Azure DevOps Product Documentation

### PR DESCRIPTION
**Pull request by mtsuji-cobalt**
### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Name | Link | Comment |

### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Name | Link | Comment |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

**Review by inikolov89 — APPROVED** [review]

**Review by eugenerevzin — APPROVED** [review]

File: content/en/Integrations/Azure DevOps/_index.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+<br>
+
+Yes. When configuring the integration for a pentest, you can set values for all standard and custom fields in the work item type template for the Project and Work Item Type selected. See: [How to set work item field values](/integrations/azure-devops/push-findings/#set-work-item-field-values).
+
+{{% /expand %}}
+<br>
+{{%expand "Can I manually create a work item for an individual finding?" %}}
+<br>
+
+No, auto-push is currently not supported.
```
Comment (grayson-cobalt) [inline content/en/Integrations/Azure DevOps/_index.md:65]: Should this be manual-push? Auto-push is the only method we support.

File: content/en/Integrations/Azure DevOps/integrate-with-azure-devops.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+1. In Cobalt, go to **Integrations** > **Azure DevOps Boards**.
+2. On the **Connection** tab, click **Add Connection**.
+3. In the **Configure Connection with Azure DevOps Boards** form of the overlay, enter:
+   - OAuth 2.0 grant type
+   - Organization: Your Azure DevOps Organization ID (`https://dev.azure.com/{yourOrganization}`)
+   - API Version: 7.0
+   - Username: Your account email address in Azure DevOps
+   - Personal Access Token: Your Azure DevOps [personal access token](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows)
+4. Click **Done**.
```
Comment (grayson-cobalt) [inline content/en/Integrations/Azure DevOps/integrate-with-azure-devops.md:21]: They need to click the "connect" button after entering their info before they click the done button.

File: content/en/Integrations/Azure DevOps/troubleshoot-azure-devops-integration.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+3. In the workflow window, select **Jobs** from the tab bar.<br><br>
+    ![Azure DevOps recipe jobs tab](/integrations/ADO-integration-check-error-jobs.png "Azure DevOps recipe jobs tab")
+
+## Configuration Forms Do Not Appear
+
+The forms do not appear when attempting to set up a connection or configure Azure DevOps for a pentest.
+
+Possible reasons:
+- Blocked cookies: If your browser is set to block all third-party cookies, you must allow cookies for `cobalt.io` by following [these instructions](https://support.google.com/chrome/answer/95647?sjid=8733712878597538106-NA#zippy=%2Callow-or-block-third-party-cookies%2Callow-third-party-cookies-for-a-specific-site). 
+- Outdated cookies: If this is your first time using the Azure DevOps integration in the Cobalt platform, you must first [clear Cobalt session cookies](/integrations/azure-devops/troubleshoot-azure-devops-integration/#clear-cobalt-session-cookies).
```
Comment (grayson-cobalt) [inline content/en/Integrations/Azure DevOps/troubleshoot-azure-devops-integration.md:35]: Users no longer have to clear their cobalt cookies before using the integrations engine for the first time :)
I deployed a technical fix for that

File: content/en/Integrations/Azure DevOps/_index.md (hunk 1 above)
Comment (eugenerevzin) [reply content/en/Integrations/Azure DevOps/_index.md:65]: Good catch.  Yes this should read "No, only auto-push is currently supported".

Comment (mtsuji-cobalt) [reply content/en/Integrations/Azure DevOps/_index.md]: Sorry that was a misinformed edit on my part.

**Review by grayson-cobalt — APPROVED** [review]

**Review by inikolov89 — APPROVED** [review]

**Pull request merged**